pyproject.toml

```diff
@@ -1,7 +1,73 @@
 [build-system]
-requires = ["setuptools >= 40.6.0", "wheel"]
+requires = ["setuptools >= 75.3.2", "wheel"]
```

Comment:
I've bumped this a bit to the latest version that support Python 3.8

Reply:
That makes sense, thanks for doing that!